app/src/main/java/com/infomaniak/drive/ui/menu/GalleryFragment.kt

```diff
@@ -195,7 +203,7 @@ class GalleryFragment : MultiSelectFragment(MATOMO_CATEGORY), NoItemsLayoutView.
             }
         }
 
-        galleryRecyclerView?.layoutManager = gridLayoutManager
+        binding.galleryRecyclerView?.layoutManager = gridLayoutManager
```

Comment:
As this was nullable, isn't there any chance that this could result in an NPE ? 
It would be safer to keep the nullability for the binding here
Either way, this null check here is useless with "binding"

Reply:
It looks like removing the null check should be fine because it's only called inside `onViewCreated` and `onConfiguraitonChanged` but looking back at where this `?`came from, it looks like this was introduced to fix something. Without much knowledge as to why this got here, we'll keep it just to be sure